src/components/Table/Table.tsx

```diff
@@ -101,9 +101,8 @@ export class Table<I extends TableDataItem = Record<string, string>> extends Rea
     };
 
     // Static methods may be used by HOCs
-    static getRowId<I extends TableDataItem>(props: TableProps<I>, item: I) {
-        const {data, getRowId} = props;
-        const index = data.indexOf(item);
+    static getRowId<I extends TableDataItem>(props: TableProps<I>, item: I, index: number) {
```

Comment:
This is a breaking change, not sure we are ready to next major now

Reply:
I can add an index as optional parameter and if it is not present fall back to the previous logic.

Reply:
I think so, in the next major we might want to make the parameter required